packages/dso-toolkit/src/components/button-row/button-row.scss

```diff
@@ -31,11 +31,11 @@
     }
   }
 
-  &.justify-end {
+  &.dso-align-right {
```

Comment:
liever niet 'align', dat betekent in flex-box de 'cross-axis', dus kan verwarrend zijn.